src/Uno.UI.Runtime.Skia.Tizen/Tizen/TizenApplicationViewExtension.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.ViewManagement;
+
+namespace Uno.UI.Runtime.Skia
+{
+	public class TizenApplicationViewExtension : IApplicationViewExtension
+	{
+		private readonly ApplicationView _owner;
+		private readonly IApplicationViewEvents _ownerEvents;
+
+		public TizenApplicationViewExtension(object owner)
+		{
+			_owner = (ApplicationView)owner;
+			_ownerEvents = (IApplicationViewEvents)owner;
+		}
+
+		public string Title { get => ""; set { } }
```

Comment:
Codacy found an issue: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5993935)